ffi_derive/src/lib.rs

```diff
@@ -132,6 +133,73 @@
 //!
 //! See `../tests/remote_types` for an example.
 //!
+//! Similarly, sometimes a type we want to expose will reference a typealias defined in a remote
+//! crate. We support that, but because the type information that backs the alias isn't available at
+//! the time procedural macros run, we require some additional configuration in both the module that
+//! defines the alias, and on the type whose fields reference the alias.
+//!
+//! ### Remote alias definitions
+//!
+//! When a module defines aliases that may be referenced by a type that derives an FFI, the
+//! `alias_resolution` attribute macro needs to be run on it in order to populate the definitions of
+//! those aliases somewhere so that we can reference them when resolving the underlying types of
+//! alias references. The macro invocation also needs to define a unique string for the module
+//! (which we refer to internally as the `resolution_key`). This will be used with a helper
+//! attribute on types that derive an FFI so that we can identify the source where their aliases are
+//! defined.
```

Comment:
pretty straightforward :D

Reply:
Super simple, right? 🙈

Reply:
I probably shouldn't still be able to see this right? Someone should probably remove me from something... 

And also, Yeah, totally simple!

Reply:
> I probably shouldn't still be able to see this right? Someone should probably remove me from something...

Thanks for the ping, @dicroce! We actually have this one open-sourced for now, so we're 👍. I got nervous there for a sec though!